src/rtk/features/apiDelegation/overviewOrg/overviewOrgSlice.ts

```diff
@@ -322,7 +338,14 @@ const overviewOrgSlice = createSlice({
       })
       .addCase(fetchOverviewOrgsOffered.rejected, (state, action) => {
         state.loading = true;
-        state.error = action.error.message ?? 'Unknown error';
+        state.error.code = String(action.payload?.response?.status) ?? 'Unknown code';
```

Comment:
Did you mean `state.error.statusCode` here? (And in the other refs to it?)
Maybe that's what makes it crash?

Reply:
Yup, good catch! Changed it now